packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,63 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+function range(file, node, source, color, CONTEXT = 2) {
```

Comment:
Either constant out of the function scope, or a regular parameter inside the function scope (if you are doing that, it would probably also be good to make these an object instead of positional arguments).